R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -82,9 +87,29 @@ TableHandle <- R6Class("TableHandle",
             arrow_tbl = self$to_arrow_table()
             return(as.data.frame(as.data.frame(arrow_tbl))) # TODO: for some reason as.data.frame on arrow table returns a tibble, not a data frame
         }
+    ),
+    active = list(
+
+        #' @description
+        #' Indicator variable for whether the table referenced by this TableHandle is static or not.
+        #' @return TRUE if the table is static, FALSE if the table is ticking.
+        is_static = function() {
+            return(private$is_static_field)
+        },
```

Comment:
I'm confused by this R -- or possibly just the documentation.  Is `is_static` a variable or a function?  From reading the code, it looks like a function, so these docs should not talk about indicator variables.  Also, it seems like the class docs for the fields should be removed to be consistent with the rest of the docs.